cmd/stdiscosrv/querysrv.go

```diff
@@ -18,7 +18,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/golang/groupcache/lru"
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
This is however how `goimports` inserts it when it adds the import, as the package name is different from the import path... I have no strong opinion either way, really.